apps/dock/dock.py

```diff
@@ -13,9 +13,33 @@ class Actions:
     def dock_send_notification(notification: str):
         """Send a CoreDock notification to the macOS Dock using SPI"""
 
+    def dock_app_expose():
+        """Activate app Exposé using the macOS Dock item for the frontmost app"""
+
 
 @ctx.action_class("user")
 class UserActions:
+    def dock_app_expose():
+        from pathlib import Path
```

Comment:
Imports should probably be at the file module level unless it's needed to break a circular import